int_tests/test_query.py

```diff
@@ -1,28 +1,97 @@
+import time
 import unittest
-import logging
 
+from vectara.core import RequestOptions
 from vectara.factory import Factory
-from vectara.corpora.client import CorporaClient
-from vectara import Vectara
+from vectara import CoreDocument, CoreDocumentPart, SearchCorporaParameters, KeyedSearchCorpus, \
+    ContextConfiguration, CustomerSpecificReranker, GenerationParameters, ModelParameters, CitationParameters, \
+    ChatParameters, QueryStreamedResponse, QueryFullResponse
 
-class FactoryTest(unittest.TestCase):
 
+class TestMultipleCorporaQuery(unittest.TestCase):
+    def setUp(self):
+        self.addCleanup(self.cleanup)
+        self.client = Factory().build()
+        self.client.corpora.create(name="test-search-1", key="test-search-1")
+        self.client.corpora.create(name="test-search-2", key="test-search-2")
+        time.sleep(30)
+        test_search_1_document = CoreDocument(
+            id="my-doc-id",
+            document_parts=[
+                CoreDocumentPart(
+                    text="Robot Utility Models are trained on a diverse set of environments and objects, and then can "
+                         "be deployed in novel environments with novel objects without any further data or training.",
+                )
+            ],
+        )
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.logger = logging.getLogger(self.__class__.__name__)
+        test_search_2_document = CoreDocument(
+            id="my-doc-id",
+            document_parts=[
+                CoreDocumentPart(
+                    text="We show that it is possible to create general Robot Utility Models with a moderate amount "
+                         "of data in the order of 1,000 demonstrations (Section 2). These RUMs achieve a 90% average "
+                         "success rate on zero-shot deployment in 25 novel environments (Section 3.1).",
+                )
+            ],
+        )
+        self.client.documents.create("test-search-1", request=test_search_1_document)
+        self.client.documents.create("test-search-2", request=test_search_2_document)
 
-    def test_query(self) -> None:
-        self.logger.info("Starting test")
+        self.search_params = SearchCorporaParameters(
```

Comment:
I would add a few more tests like:
1. lexical_interpolation values 0, 0.1, 1.0 (as right now)
2. More reranking tests (in addition to rnk_272725719) - MMR, UDF and chain
3. Tests for other citations options
4. Tests for other languages (response_langugage)
5. Tests for FCS on and off
6. Tests for various prompt (sumamrizer) values and custom prompt text.